src/daisy_seed.h

```diff
@@ -149,6 +149,22 @@ class DaisySeed
     */
     using Log = Logger<LOGGER_INTERNAL>;
 
+    /** Internal indices for DaisySeed-equivalent devices 
+     *  This shouldn't have any effect on user-facing code,
+     *  and only needs to be checked to properly initialize
+     *  the onboard-circuits.
+    */
+    enum BoardVersion
```

Comment:
can we make this an `enum class` - unless casting to int is done regularily, enum class is typically the more strong typed option

Reply:
yeah, actually had the same thought after I pushed. I don't see ever needing to cast as int. So will totally do this.